### PR TITLE
fix: delete unused var and const

### PR DESCRIPTION
**Pull request by hwdef**
### 1. Why is this pull request needed and what does it do?
delete unused var and const
### 4. Does this introduce a backward incompatible change or deprecation?
NO

### CODE REVIEW

File: plugin/dnssec/handler.go [hunk 1]
```diff
@@ -55,13 +55,6 @@ var (
 		Help:      "The number of elements in the dnssec cache.",
 	}, []string{"server", "type"})
 
-	cacheCapacity = prometheus.NewGaugeVec(prometheus.GaugeOpts{
```
Comment (miekg) [inline plugin/dnssec/handler.go:58]: this needs to be removed from the README as well. (no-one ever complained this wasn't exported, so I'm OK with removing it)

**Comment by miekg** [comment]
otherwise lgtm

File: plugin/dnssec/handler.go (hunk 1 above)
Comment (hwdef) [reply plugin/dnssec/handler.go:58]: Thanks.

Comment (hwdef) [reply plugin/dnssec/handler.go:58]: @miekg 
I can not find it in README,
Can you help me find out it?
Thank you!

Comment (miekg) [reply plugin/dnssec/handler.go:58]: ah, it's not there. my mistake

**Pull request merged**